sqlx-postgres/src/connection/describe.rs

```diff
@@ -492,13 +488,19 @@ fn visit_plan(plan: &Plan, outputs: &[String], nullables: &mut Vec<Option<bool>>
 }
 
 #[derive(serde::Deserialize)]
+#[serde(untagged)]
 enum Explain {
     /// {"Plan": ...} -- returned for most statements
-    Plan(Plan),
+    QueryPlan(QueryPlan),
     /// The string "Utility Statement" -- returned for
     /// a CALL statement
-    #[serde(rename = "Utility Statement")]
-    UtilityStatement,
+    UtilityStatement(String),
```

Comment:
So this enum variant is never used? Its definiton was changed, but I see no other code that had to be updated because of that.

Reply:
correct, that's my understanding how it was working before my changes as well

Reply:
so apparently it's required for compile-time checks agains DB - to match string:
```
error: error occurred while decoding column 0: data did not match any variant of untagged enum Explain at line 3 column 1
Error:    --> tests/postgres/macros.rs:103:15
    |
103 |     let row = sqlx::query!(r#"CALL forty_two(null)"#)
    |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: this error originates in the macro `$crate::sqlx_macros::expand_query` which comes from the expansion of the macro `sqlx::query` (in Nightly builds, run with -Z macro-backtrace for more info)

error: could not compile `sqlx` (test "postgres-macros") due to previous error
Error: warning: build failed, waiting for other jobs to finish...
Error: The process '/home/runner/.cargo/bin/cargo' failed with exit code 101
```
https://github.com/launchbadge/sqlx/actions/runs/5632237064/job/15260004403?pr=2626